fabtests/common/shared.c

```diff
@@ -119,7 +119,7 @@ struct fi_rma_iov remote;
 
 struct ft_opts opts;
 
-struct test_size_param test_size[] = {
+struct test_size_param all_test_size[] = {
```

Comment:
In the future, it would be better to separate out name changes like this as a separate patch.  Don't worry about that change for this.

Reply:
Actually, let's rename this to def_test_sizes rather than all.